modules/preview-react/form-field/lib/FormField.tsx

```diff
@@ -90,7 +99,11 @@ export const FormField = createContainer('div')({
   const layoutProps = useFormFieldOrientation(model.state.orientation);
 
   return (
-    <Flex as={Element} {...layoutProps} {...mergeStyles(elemProps, formFieldBaseStyles)}>
+    <Flex
+      as={Element}
+      {...layoutProps}
+      {...mergeStyles(elemProps, formFieldBaseStyles({grow: model.state.grow}))}
+    >
```

Comment:
I think `layoutProps` should be a modifier instead. Orientation is `'horizontal' | 'vertical'`. As long as we're moving toward static styling, we should also remove dynamic style props in favor of modifiers.